resources/todomvc/vanilla-examples/javascript-es5/src/store.js

```diff
@@ -56,6 +56,7 @@
                         return false;
                 }
 
+
```

Comment:
the ember cleanup pr has a fix for this double space... 
https://github.com/WebKit/Speedometer/pull/119

Reply:
Thanks - I'll revert this file change